int32 accX
int32 accY
int32 accZ
float32 latitude
float32 longitude
float32 heading
float32 speed
float32 altitude